processes/fulfill_url_manifest.py

```diff
@@ -0,0 +1,166 @@
+import json
+import os
+import copy
+import logging
+from botocore.exceptions import ClientError
+
+from .core import CoreProcess
+from datetime import datetime, timedelta, timezone
+from model import Link
+from logger import createLog
+
+logger = createLog(__name__)
+
+class Fulfill_Process(CoreProcess):
+
+    def __init__(self, *args):
+        super(Fulfill_Process, self).__init__(*args[:4])
+
+        self.fullImport = self.process == 'complete' 
+        self.startTimestamp = None
+
+        # Connect to database
+        self.generateEngine()
+        self.createSession()
+
+        # S3 Configuration
+        self.s3Bucket = os.environ['FILE_BUCKET']
+        self.host = os.environ['DRB_API_HOST']
+        self.prefix = 'manifests/UofM/'
+        self.createS3Client()
+
+    def runProcess(self):
+        if self.process == 'daily':
+            startTimeStamp = datetime.now(timezone.utc) - timedelta(days=1)
+            self.get_manifests(startTimeStamp)
+        elif self.process == 'complete':
+            self.get_manifests()
+        elif self.process == 'custom':
+            time_stamp = self.ingestPeriod
+            startTimeStamp = datetime.strptime(time_stamp, '%Y-%m-%dT%H:%M:%S')
+            self.get_manifests(startTimeStamp)
+
+    def get_manifests(self, startTimeStamp=None):
```

Comment:
startTimeStamp -> start_time_stamp

Reply:
the comment below is explaining further what this function does. Can we rename this function in order to remove the comment?

Reply:
Sure. I can rename it to get_manifest_batches

Reply:
We are updating the manifests in this function though. I recommend either renaming it so it is clear that the function is updating or you split the function up - 1 get function that returns the batches and another that updates the batches.